tests/baselines/reference/wellKnownSymbolExpando.types

```diff
@@ -8,7 +8,8 @@ function f() {}
 f[Symbol.iterator] = function() {}
 >f[Symbol.iterator] = function() {} : () => void
 >                                   : ^^^^^^^^^^
->f[Symbol.iterator] : any

```

Comment:
the issue was already included in the existing test suite - but it wasn't overly apparent because this test was not using strict mode so the implicit any was not reported separately